app/src/main/java/com/uberspot/a2048/MainActivity.java

```diff
@@ -36,12 +32,11 @@
     private long mLastBackPress;
     private static final long mBackPressThreshold = 3500;
     private static final String IS_FULLSCREEN_PREF = "is_fullscreen_pref";
-    private static boolean DEF_FULLSCREEN = true;
```

Comment:
Why remove this and hardcode to "true" ? It's a convenient place to change this as an option. Can we reset this?